src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php

```diff
@@ -137,18 +137,18 @@ protected function createConfigurationDefinition()
             'array_key_exists',
             'empty',
             'file_exists',
-            'is_infinite',
-            'is_nan',
-            'is_null',
             'is_array',
             'is_bool',
             'is_boolean',
             'is_callable',
             'is_double',
             'is_float',
+            'is_infinite',
             'is_int',
             'is_integer',
             'is_long',
+            'is_nan',
+            'is_null',
```

Comment:
what about calling sort on it, to actually ensure values in readme are sorted, even if this list would not be ?

Reply:
Can't we replace this hardcoded array with `array_keys(self::$fixMap)`?

Reply:
no, next time someone will update possible options, he will not-willingly update default value as well